endor/README.md

```diff
@@ -90,3 +90,41 @@ sequelize.User.findAll({
   console.error(err);
 });
 ```
+
+##Authentication
+
+###Basic-Auth
+Uses the user's username and password in the Authentication header to authenticate
+the user.
+
+###Client-Basic
+Uses the client's clientId and secret like a username and password to authenticate and
+to protect a user's credentials.
+
+###Bearer
+A token is used to authenticate the user.
```

Comment:
Your headers are not coming out properly in the README.md file, there needs to be a space between the # group and the heading. For example 

`# Authentication` creates
# Authentication